From: Lorenzo Stoakes <lstoakes@gmail.com>
Subject: mm: remove unused alloc_pages_bulk_list()
Date: Sat, 17 Dec 2022 00:15:54 +0000

This function is not referenced anywhere else in the kernel, remove it.

Link: https://lkml.kernel.org/r/20221217001554.554913-1-lstoakes@gmail.com
Signed-off-by: Lorenzo Stoakes <lstoakes@gmail.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 include/linux/gfp.h |    7 -------
 1 file changed, 7 deletions(-)

--- a/include/linux/gfp.h~mm-remove-unused-alloc_pages_bulk_list
+++ a/include/linux/gfp.h
@@ -188,13 +188,6 @@ unsigned long alloc_pages_bulk_array_mem
 				unsigned long nr_pages,
 				struct page **page_array);
 
-/* Bulk allocate order-0 pages */
-static inline unsigned long
-alloc_pages_bulk_list(gfp_t gfp, unsigned long nr_pages, struct list_head *list)
-{
-	return __alloc_pages_bulk(gfp, numa_mem_id(), NULL, nr_pages, list, NULL);
-}
-
 static inline unsigned long
 alloc_pages_bulk_array(gfp_t gfp, unsigned long nr_pages, struct page **page_array)
 {
_
